<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QWidget" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>210</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="privateKeyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>私钥:</string>
   </property>
  </widget>
  <widget class="QLabel" name="publicKeyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>公钥:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="publicKeyAddr">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="privateKeyAddr">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectUserButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPublicKeyButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPrivateKeyButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择...</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPasswordButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
